action.yml

```diff
@@ -99,10 +99,10 @@ runs:
             done
           fi
           if [ ! "${{ inputs.included_files }}" = "" ]; then
-            solsta_extra_args+="--include=\"${{ inputs.included_files }}\" "
+            solsta_extra_args+="--include=${{ inputs.included_files }} "
```

Comment:
I'm going to take your word and approve it, but I don't know why you wouldn't want quotes around an argument.

Reply:
I was surprised too, but I'd never tried that with argparse before, and gammy said it worked w/o.  Turns out there was some confusion on that point.